### PR TITLE
audio accessibility support

### PR DESCRIPTION
**Pull request by kristofmuhi**
### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/mobile-apps/features/audio-capture.md [hunk 1]
```diff
@@ -70,23 +70,27 @@ If your apps's manifest.xml file defines `android:allowAudioPlaybackCapture="fal
 
 :::
 
-## Using TalkBack on Android Private Devices
+## Enabling TalkBack/Voiceover on your Devices
```
Comment (yuyi-sl) [inline docs/mobile-apps/features/audio-capture.md:73]: VoiceOver with capital O

File: docs/mobile-apps/features/audio-capture.md [hunk 2]
```diff
@@ -70,23 +70,27 @@ If your apps's manifest.xml file defines `android:allowAudioPlaybackCapture="fal
...
 
-To utilize TalkBack on your Android private device, make sure you have a device with TalkBack enabled (Android 10 and above, Google Pixel or Samsung devices). Follow these steps:
+To utilize TalkBack on your iOS/Android devices, make sure you have access to supported devices, We support iOS 13 and above, and Android 10 and above.
```
Comment (yuyi-sl) [inline docs/mobile-apps/features/audio-capture.md:75]: ... utilize VoiceOver/TalkBack on your ...
Maybe keep the order for Android or iOS first the same as in the headline

Comment (sakhisheikh) [reply docs/mobile-apps/features/audio-capture.md:75]: Should we `Audio Accessibility` term to cover both `VoiceOver/TalkBack`.

File: docs/mobile-apps/features/audio-capture.md [hunk 3]
```diff
@@ -70,23 +70,27 @@ If your apps's manifest.xml file defines `android:allowAudioPlaybackCapture="fal
...
+2. Open the **Device Settings** from the left side toolbar.
 
-3. Select **SL_TalkBack Application**: Look for the SL_TalkBack application featuring the Sauce Labs logo.
+3. Select and toggle **Talkback** or **Voiceover**
 
-4. Click **User Service**.
+4. Skip with OK or Cancel the native Talkback/Voiceover guide.
 
-5. Enable **Allow Talkback**.
+5. On Android you can use the Arrow keys (Up-Down-Left-Right) to navigate through the elements, and by hitting the ENTER key, you are able to activate a specific element.
```
Comment (sakhisheikh) [inline docs/mobile-apps/features/audio-capture.md:88]: should we say  " key modifier shortcuts" are not supported at the moment.

File: docs/mobile-apps/features/audio-capture.md (hunk 1 above)
Comment (kristofmuhi) [reply docs/mobile-apps/features/audio-capture.md:73]: 🤦 good catch

File: docs/mobile-apps/features/audio-capture.md (hunk 3 above)
Comment (kristofmuhi) [reply docs/mobile-apps/features/audio-capture.md:88]: i would only call out what is supported. 
We will improve it with the UX, so it is clear what do we support

File: docs/mobile-apps/features/audio-capture.md (hunk 2 above)
Comment (kristofmuhi) [reply docs/mobile-apps/features/audio-capture.md:75]: it should cover. but i wanna make it more easily discoverable. i wanna call out everywhere Talkback/Voiceover

**Pull request merged**